cmd/build/command.go

```diff
@@ -80,22 +89,21 @@ const (
 )
 
 // Build build and optionally push a Docker image
-func Build(ctx context.Context, at analyticsTrackerInterface) *cobra.Command {
-
+func Build(ctx context.Context, oktetoLogger *io.IOController, at analyticsTrackerInterface) *cobra.Command {
```

Comment:
```suggestion
func Build(ctx context.Context, ioCtrl *io.IOController, at analyticsTrackerInterface) *cobra.Command {
```
logger is confused I think, logger is part of the ioctrl, no?